docs/rules/DisableSyntax.md

```diff
@@ -11,15 +11,16 @@ rule.
 Example:
 
 ```scala
-MyCode.scala:7: error: [DisableSyntax.xml] xml is disabled.
-  <a>xml</a>
-  ^
+MyCode.scala:7: error: [DisableSyntax.xml] xml literals is disabled.
+  var x = 2
+  ^^^
```

Comment:
this looks wrong (message vs actual code)

Reply:
Good catch! I removed the var example, `throw` is enough to get the idea. Thanks